### PR TITLE
WIP Testing CI stuff, also testing danger-js

### PR DESCRIPTION
**Pull request by sadasant**
We have danger-js! :D

### CODE REVIEW

**Comment by sadasant** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>No Changelog changes!</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by sadasant** [comment]
Yup we're not doing danger-js 😱 imagine anyone submitting a PR using malicious code with my token 👀💧

**Comment by sadasant** [comment]
This is ready to be merged!

**Comment by sadasant** [comment]
Wait, I'll fix the changelog and version requirements with bashy lines

File: circle.yml [hunk 1]
```diff
@@ -22,7 +18,12 @@ test:
   pre:
     - git remote set-branches --add origin master
     - git fetch
+    # Eslint changed files only
     - LIST=$(git diff-index --name-only origin/master | grep .*\\.js | grep -v json | grep -v eslint | tr '\n' ' '); if [ "$LIST" ]; then eslint $LIST; fi
+    # Check if CHANGELOG.md was updated
+    - git diff --name-only origin/master | grep CHANGELOG
+    # Check if package.json was updated
+    - git diff --name-only origin/master | grep package.json
```
Comment (sadasant) [inline circle.yml]: @daedalus28 do you like this? 👆

**Comment by decrappifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Please add a changelog entry for your changes.
Kindly,</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by decrappifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Please add a changelog entry for your changes.</td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>
  
Kindly,</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by decrappifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Please add a changelog entry for your changes.</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**